ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Version used:
+* Browser Name and version:
+* Operating System and version (desktop or mobile):
+* Link to your project:
+
+## Context
+<!--- For bugs, Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+<!--- How has this bug affected you? -->
+<!--- If you have errors or a stacktrace in the console, here is a good place to put it. -->
+<!--- Please consider using a pastebin service or create a gist.-->
+<!--- Some screenshots are always greatly appreciated here! -->
+
+<!--- For improvement suggestions, please tell us what you are trying to accomplish -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
+2.
+3.
+4.
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what you think should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Actual Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Cause/Fix/Solution
+<!--- Not obligatory, but suggest a cause or reason for the bug if relevant, -->
+<!--- or ideas how to implement the addition or change -->
+<!--- In any case, please consider opening a PR to become an iTowns developer! -->
```

Comment:
nit: “…consider opening a PR **and** become an iTowns developer” ?

Reply:
Yep, done :-)